packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/index.md

```diff
@@ -43,56 +45,39 @@ The SDK represents the sign-in flow using a number of different objects:
-- **Field:** Represents a UI element, either a static item, such as a label, or user input, such as a selection list. It includes properties for state information, such as whether the associated value is required. Properties also store the current value of user input field, such as the string for an OTP or the selected choice. Options, or lists of choices, are represented by a collection of fields. A field may contain a form that contains more fields.
+- **Field:** Represents a UI element, either a static item, such as a label, or user input, such as a selection list. It includes properties for state information, such as whether the associated value is required. A property also stores the current value of a user input field, such as the string for an OTP or the selected choice. A collection of fields, or **Options**, represents a list of choices. A field may contain a form that contains more fields.
 - **Form:** Contains the fields that represent the user action for a remediation.
-- **Configuration:** Contains the settings used by the SDK to connect to your Okta org app integration.
-- **Client:** Represents the session during the sign-in flow.
+- **InteractionCodeFlow:** Represents the session during the sign-in flow. In the Android SDK it includes the function for requesting the next step in the flow.
 
 
 ## Objects and the flow
 
-<div class="three-quarter">
-
-!["A diagram that shows the main objects associated with each step in the sign-in flow."](/img/mobile-sdk/mobile-idx-objects-and-flow.png)
-
-</div>
+<StackSnippet snippet="objectsandflow" />
 
-The main objects associated with each step in the flow are:
+## Manage the sign-in flow
 
-| Sign-in step                       | Objects                         |
-| :--------------------------------- | :------------------------------ |
-| Initialize SDK                     | Configuration <br/> Client      |
-| Request initial step               | Client                          |
-| Receive step                       | Response                        |
-| Check completion, cancel, or error | Response <br/> Remediation      |
-| Gather user input                  | Remediation <br/> Authenticator |
-| Send input                         | Remediation                     |
-| Done                               | Response                        |
+To create an object that manages the sign-in flow, include the following functionality:
 
-
-## Common parts of the flow
-
-Some parts of the sign-in flow are the same:
-
-- Add the SDK to your app.
-- Configure the SDK.
-- Initialize the client and start the flow.
+- Initialize the flow.
+- Start the sign-in.
 - Process the response.
 - Request a token.
 - Sign the user out.
 
 ### Add the SDK
 
+Before you can implement the manager you need to add the SDK to your project.
+
 <StackSnippet snippet="adddependency" />
```

Comment:
Optional. In add dependency for Android it makes more sense to link to the list of actual releases in the SDK repo rather than the list of all tags. - https://github.com/okta/okta-idx-android/releases

Reply:
I'm going to leave it as tags as that was originally written by someone who knows Android/Kotlin. I'm assuming that a list of tags has some meaning for Android... I also looked at both pages (tag list and release list) and the tag list looked cleaner ;-).